src/helpers.js

```diff
@@ -111,12 +117,13 @@ export async function promptUser(packageName: string, dependencies: Array<Depend
       }, {
         text: 'Never',
         onDidClick: () => {
-          configFile.append('ignored', packageName)
+          ignoredPackages.push(packageName)
+          atom.config.set('sb-package-deps.ignored', ignoredPackages)
           if (!shownStorageInfo) {
             shownStorageInfo = true
             atom.notifications.addInfo('How to reset package-deps memory', {
               dismissable: true,
-              description: `If you ever wish to change the packages package-deps never installs, please modify ${tildify(configPath)}`,
+              description: `If you ever wish to change the packages package-deps never installs, please modify ${tildify(configPath)}.sb-package-deps.ignored`,
```

Comment:
There is no separate file for this, all settings using the API are within the `~/.atom/config.cson` file.